package.json

```diff
@@ -9,21 +9,28 @@
     "build:demo": "vue-cli-service build --dest _site/ demo/main.js",
     "lint:style": "stylelint 'src/**/*.vue' 'demo/**/*.vue'"
   },
-  "main": "dist/girder.umd.min.js",
+  "browser": "dist/girder.umd.min.js",
+  "module": "src/index.js",
+  "main": "src/index.js",
+  "types": "types/index.d.ts",
```

Comment:
"browser" will allow unpkg to continue serving the UMD while making esm imports more concise/normal, which is what we all use anyway.